queries.sql

```diff
@@ -61,4 +61,60 @@ SELECT name, full_name FROM animals JOIN owners ON animals.owner_id = owners.id
 
 SELECT full_name, name FROM animals LEFT JOIN owners ON owner_id = owners.id WHERE owners.id=5 and escape_attempts=0;
 
-SELECT full_name , COUNT(animals.name) AS all_animals FROM owners JOIN animals ON owner_id=owners.id GROUP BY owners.id ORDER BY all_animals DESC LIMIT 1;
\ No newline at end of file
+SELECT full_name , COUNT(animals.name) AS all_animals FROM owners JOIN animals ON owner_id=owners.id GROUP BY owners.id ORDER BY all_animals DESC LIMIT 1;
+
+--day4 project
+
+--Who was the last animal seen by William Tatcher?
+SELECT animal.id,  animal.name, vt.date_of_visit FROM animals 
+animal JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```

Comment:
Please correct your request, you add animal twice : 

```
SELECT animal.id,  animal.name, vt.date_of_visit FROM **animals 
animal** JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```
Correct request : 
```
SELECT animal.id,  animal.name, vt.date_of_visit FROM **animals ** JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```

Reply:
@jaflih its not animal twice its animal and the table name of the table animals with s
its not the same